prow/plugins/cla/cla.go

```diff
@@ -174,3 +182,79 @@ func handle(gc gitHubClient, log *logrus.Entry, se github.StatusEvent) error {
 	}
 	return nil
 }
+
+func handleCommentEvent(pc plugins.PluginClient, ce github.ReviewCommentEvent) error {
+	return handleComment(pc.GitHubClient, pc.Logger, &ce)
+}
+
+func handleComment(gc gitHubClient, log *logrus.Entry, e *github.ReviewCommentEvent) error {
+	// Only consider open PRs and new comments.
+	if e.PullRequest.State != "open" || e.Action != github.ReviewCommentActionCreated {
+		return nil
+	}
+	// If we create a "/check-cla comment, check for the cla in commit statuses, add cla label if necessary.
+	checkCLA := false
```

Comment:
This seems unnecessary (either it is true or we return).

Reply:
done.